### PR TITLE
fix(Menu): fix icon and submenuIndicator alignment for non-v9 icons

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Current Behavior

The menu item does not align the icon slot and submenuIndicator slot content with the menu item text.

menu-before-alignment

submenu-before-alignment

> These were created by modifying the associated stories to use the v8 icons and spans with plain text as the icon or submenuIndicator.

## New Behavior

Menu item aligns icon and submenuIndicator slot content using flex just like button does.

menu-after-alignment

submenu-after-alignment

## Related Issue(s)

This was discovered as part of writing the ContextualMenuButton shim for the v8 -> v9 upgrade.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                                                                Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `171.967 kB`<br />`48.194 kB` |   `172.3 kB`<br />`48.272 kB` | `333 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`78 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          |  `103.43 kB`<br />`31.858 kB` | `103.763 kB`<br />`31.935 kB` | `333 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`77 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `105.785 kB`<br />`32.219 kB` | `106.118 kB`<br />`32.298 kB` | `333 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`79 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                  |         Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `32.526 kB`<br />`10.645 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/2fe4ead4770b6292061f54ddc1749178443f6f5a'>2fe4ead4770b6292061f54ddc1749178443f6f5a</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MenuButton-mount_base.html">1799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MenuButton-mount.html">1813</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MenuButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Avatar-mount_base.html">1134</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Avatar-mount.html">1181</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/BaseButton-mount_base.html">1225</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/BaseButton-mount.html">1232</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Breadcrumb-mount_base.html">3175</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Breadcrumb-mount.html">3167</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ButtonNext-mount_base.html">720</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ButtonNext-mount.html">739</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Checkbox-mount_base.html">1835</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Checkbox-mount.html">1830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/CheckboxBase-mount_base.html">1583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/CheckboxBase-mount.html">1613</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ChoiceGroup-mount_base.html">5239</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ChoiceGroup-mount.html">5197</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ComboBox-mount_base.html">1303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ComboBox-mount.html">1250</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/CommandBar-mount_base.html">11500</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/CommandBar-mount.html">11404</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ContextualMenu-mount_base.html">21758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ContextualMenu-mount.html">21682</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DefaultButton-mount_base.html">1439</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DefaultButton-mount.html">1444</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DetailsRow-mount_base.html">4295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DetailsRow-mount.html">4257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DetailsRowFast-mount_base.html">4247</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DetailsRowFast-mount.html">4351</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DetailsRowNoStyles-mount_base.html">4135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DetailsRowNoStyles-mount.html">4162</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Dialog-mount_base.html">2606</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Dialog-mount.html">2615</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DocumentCardTitle-mount_base.html">405</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/DocumentCardTitle-mount.html">409</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Dropdown-mount_base.html">3735</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Dropdown-mount.html">3753</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderNext-mount_base.html">2288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderNext-mount.html">2262</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderWithTheme-mount_base.html">392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderWithTheme-mount.html">391</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">328</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderWithTheme-virtual-rerender.html">349</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">442</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">412</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FocusTrapZone-mount_base.html">2171</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FocusTrapZone-mount.html">2151</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FocusZone-mount_base.html">2103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/FocusZone-mount.html">2161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/IconButton-mount_base.html">2095</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/IconButton-mount.html">2102</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Label-mount_base.html">603</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Label-mount.html">647</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Layer-mount_base.html">3383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Layer-mount.html">3383</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Link-mount_base.html">730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Link-mount.html">769</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MakeStyles-mount_base.html">2031</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MakeStyles-mount.html">1992</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MenuButton-mount_base.html">1799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MenuButton-mount.html">1813</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MenuButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MessageBar-mount_base.html">2338</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/MessageBar-mount.html">2352</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Nav-mount_base.html">3716</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Nav-mount.html">3748</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/OverflowSet-mount_base.html">1409</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/OverflowSet-mount.html">1386</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Panel-mount_base.html">2478</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Panel-mount.html">2543</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Persona-mount_base.html">1183</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Persona-mount.html">1154</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Pivot-mount_base.html">1764</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Pivot-mount.html">1795</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/PrimaryButton-mount_base.html">1582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/PrimaryButton-mount.html">1599</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Rating-mount_base.html">8349</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Rating-mount.html">8371</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SearchBox-mount_base.html">1648</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SearchBox-mount.html">1608</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Shimmer-mount_base.html">2843</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Shimmer-mount.html">2892</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Slider-mount_base.html">2263</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Slider-mount.html">2262</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SpinButton-mount_base.html">5657</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SpinButton-mount.html">5602</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Spinner-mount_base.html">678</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Spinner-mount.html">701</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SplitButton-mount_base.html">3561</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SplitButton-mount.html">3649</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Stack-mount_base.html">793</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Stack-mount.html">768</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/StackWithIntrinsicChildren-mount_base.html">2726</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/StackWithIntrinsicChildren-mount.html">2678</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/StackWithTextChildren-mount_base.html">5838</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/StackWithTextChildren-mount.html">5758</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SwatchColorPicker-mount_base.html">12556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/SwatchColorPicker-mount.html">12582</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/TagPicker-mount_base.html">2997</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/TagPicker-mount.html">2985</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/TeachingBubble-mount_base.html">14035</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/TeachingBubble-mount.html">14256</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Text-mount_base.html">679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Text-mount.html">692</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/TextField-mount_base.html">1736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/TextField-mount.html">1746</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ThemeProvider-mount_base.html">1504</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ThemeProvider-mount.html">1513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ThemeProvider-virtual-rerender_base.html">885</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ThemeProvider-virtual-rerender.html">897</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2208</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2164</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Toggle-mount_base.html">1080</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/Toggle-mount.html">1100</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/buttonNative-mount_base.html">375</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21949/perf-test/buttonNative-mount.html">370</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by ling1726** [comment]
I am surprised that this did not break any vr-test, could you please add a new test for custom icons ?

File: change/@fluentui-react-menu-1918e3b8-f0c6-43a7-982c-f778eef4ecf1.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Updated icon and submenuIndicator styles to align with menu item text",
```
Comment (ling1726) [inline change/@fluentui-react-menu-1918e3b8-f0c6-43a7-982c-f778eef4ecf1.json:3]:
```suggestion
  "comment": "fix(MenuItem): Alignment for non v9 icons",
```

**Comment by GeoffCoxMSFT** [comment]
> I am surprised that this did not break any vr-test, could you please add a new test for custom icons ?

Done :)

**Review by ling1726 — APPROVED** [review]
Thanks for writing the VR tests :)

**Pull request merged**